sweepbatcher/sweep_batch_test.go

```diff
@@ -40,6 +44,15 @@ func TestConstructUnsignedTx(t *testing.T) {
 	p2trPkScript, err := txscript.PayToAddrScript(p2trAddress)
 	require.NoError(t, err)
 
+	change1 := &wire.TxOut{
+		Value:    100_000,
+		PkScript: p2trPkScript,
```

Comment:
Please use another pkscript here, because `p2trPkScript` is used as the main output by the tests, so they won't be distinguished by the test if they are confused (e.g. the order is broken).